### ISSUE TITLE
OSX - Tabpage problem with tabheader highlight

### ISSUE THREAD

**Issue by TomQv** [picoe/Eto#1308]
On OSX only the first TabControl is working correctly.

The second TabControl has this problem:

The first TabHeader always stays highlighted.
When you select another tab, the tabheader-highlight, the blue backgroundcolor, is not moving to the selected tab.

I'm still on 6 month old source code.
Just asking, if this was maybe addressed in the meanwhile?

**Comment by cwensley** [comment]
Hey @TomQv, probably not, I do not recall this issue..   Do you have screenshots and/or code that demonstrates this?

**Comment by TomQv** [comment]
No, will try to debug the issue tomorrow, assuming some static properties or fields, which just stays on the first instance of the tabcontrol,  guessing...

**Comment by TomQv** [comment]
Can't reproduce this with EtoTest.
So its most probably my code...

**Comment by TomQv** [comment]
The problem is TabControl.BackgroundColor.
Don't touch this property on Mac, it messes up rendering of TapPage-headers.

**Comment by cwensley** [comment]
@TomQv thanks for figuring out what caused the issue, it is very helpful!

**Comment by TomQv** [comment]
I didn't go further to find a fix or workaround.
When I look at the code in eto.mac-namespace I hardly understand a thing, too complicated this mac-stuff.
Anyway, for me its fine now, I don't need to change BackgroundColor.
